app/uk/gov/hmrc/apiplatformmicroservice/apidefinition/models/ApiDefinition.scala

```diff
@@ -24,6 +24,7 @@ import uk.gov.hmrc.apiplatform.modules.applications.domain.models.ApplicationId
 
 case class ApiDefinition(
     serviceName: String,
+    serviceBaseUrl: String,
```

Comment:
Is this ever returned from the API call and would we want to exclude it from the JSON writer?

Reply:
It is returned now - it needs to be added to align what comes back from API-Definition and APM - previously GK was using a mapped structure from APM but we need the api-definition case class for what this ticket solves (but not this field other than for consistent case class).  Ideally we'd have a consistent library for api shape and use it in API-Definition, APM, GK and others.